src/WebJobs.Script.WebHost/WebHostServiceCollectionExtensions.cs

```diff
@@ -93,17 +93,6 @@ public static void AddWebJobsScriptHost(this IServiceCollection services, IConfi
                 }
             });
 
-            //services.AddSingleton<ILoggerProviderFactory, WebHostLoggerProviderFactory>();
-
-            // TODO: DI (FACAVAL) Removed the previous workaround to pass a logger factory into the host resolver
-            // this is no longer needed, but we need to validate log output.
-            // Remove the need to have the WebHostResolver
-            //services.AddSingleton<WebHostResolver>();
```

Comment:
`WebHostResolver` removed, most logic replaced by the `WebJobsScriptHostService`